conda/environments/deployment_docs.yml

```diff
@@ -1,18 +1,18 @@
 name: deployment-docs-dev
 channels:
+  - nodefaults
```

Comment:
See https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-file-manually

> *You can exclude the default channels by adding `nodefaults` to the channels list.*

This will force the entire solve to be done with only `conda-forge`. That's generally faster than mixing multiple channels, because there's less repodata to download and fewer candidates packages to consider when trying to solve.